packages/server/src/workers/subscription.ts

```diff
@@ -55,7 +57,7 @@ export function initSubscriptionWorker(config: MedplumRedisConfig): void {
   queue = new Queue<SubscriptionJobData>(queueName, {
     ...defaultOptions,
     defaultJobOptions: {
-      attempts: 18, // 1 second * 2^18 = 73 hours
+      attempts: MAX_JOB_ATTEMPTS, // 1 second * 2^18 = 73 hours
```

Comment:
This will put a limit on the value that the subscription resource can have for max retries as well